src/database/mod.rs

```diff
@@ -578,12 +578,31 @@ impl Database {
         Ok(result
             .into_iter()
             .map(|row| DeletionEntry {
-                leaf_index: row.get::<i64, _>(0),
+                leaf_index: row.get::<i64, _>(0) as usize,
                 commitment: row.get::<Hash, _>(1),
             })
             .collect::<Vec<DeletionEntry>>())
     }
 
+    /// Remove a list of entries from the deletions table
+    pub async fn remove_deletions(&self, commitments: Vec<Hash>) -> Result<(), Error> {
```

Comment:
Could you write a test for this function?

I have a feeling you'd need `where commitment in $1` if you're using `push_values`